crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -1517,7 +1528,8 @@ impl<RpcMiddleware> RpcServerConfig<RpcMiddleware> {
                                 .or(modules.ws.as_ref())
                                 .map(RpcRequestMetrics::same_port)
                                 .unwrap_or_default(),
-                        ),
+                        )
+                        .layer(RpcRequestRateLimiter::new(rt.clone(), 100))
```

Comment:
these `100` should be replaced with something more configurable, but I'm not sure where to store these values.
ie. global const, or configurable value